docs/decision_record.md

```diff
@@ -6,6 +6,24 @@
 
 [TOC]
 
+## 2022-05-11 - Sequence digest recommendation
+
+### Decision
+
+The GA4GH digest will be used as our default sequence checksum derived identifier instead of an MD5 generated one.
+
+### Rationale 
+
+The GA4GH digest was created as part of the [Variation Representation Specification standard](https://vrs.ga4gh.org/en/stable/impl-guide/computed_identifiers.html). This document included a way of creating identifiers to be used with sequences e.g. ACGT results in the identifier `ga4gh:SQ.aKF498dAxcJAqme6QYQ7EZ07-fiw8Kw2`. The scheme uses the [`sha512t24u` function](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239883) to create a base64 URL-safe representation of a sha512 digest. Adopting this standard ensures sequence collections remains inline with newer standards within the GA4GH ecosystem.
```

Comment:
I would clarify the example like this:
```suggestion
The GA4GH digest was created as part of the [Variation Representation Specification standard](https://vrs.ga4gh.org/en/stable/impl-guide/computed_identifiers.html). This document included a way of creating identifiers to be used with sequences e.g. ACGT is digested as `aKF498dAxcJAqme6QYQ7EZ07-fiw8Kw2` and result in the identifier `ga4gh:SQ.aKF498dAxcJAqme6QYQ7EZ07-fiw8Kw2` once prefixed. The scheme uses the [`sha512t24u` function](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239883) to create a base64 URL-safe representation of a sha512 digest. Adopting this standard ensures sequence collections remains inline with newer standards within the GA4GH ecosystem.
```

Reply:
@andrewyatz I think we should make this ADR only about algorithm, and not about identifier construction at all, which is handled in #37

Reply:
@nsheff I agree! We should in that case make sure that we use the term `digest` and not `identifier` everywhere.